docs/src/main/sphinx/connector/hive.rst

```diff
@@ -400,6 +400,36 @@ Property Name                                      Description
                                                    not make smart decisions about the query plan.
 ================================================== ============================================================ ============
 
+Storage table properties
```

Comment:
I think just "Table properties" might be better

Reply:
Yes. These are general table properties that apply to all tables - not just the backing tables for a materialized view.

@rosewms You can see the available ones by querying `SELECT * FROM system.metadata.table_properties` (documented on https://trino.io/docs/current/sql/create-table.html#description).
That won't list "all possible" though - it lists according to how the Hive connector is configured.

A note linking to (or mentioning) how to get an exhaustive list would be very useful too.